RecoParticleFlow/PFClusterProducerAlpaka/plugins/BuildFile.xml

```diff
@@ -0,0 +1,35 @@
+<library name="RecoParticleFlowPFClusterProducersPlugins" file="*.cc">
+  <use name="FWCore/Framework"/>
+  <use name="FWCore/ParameterSet"/>
+  <use name="FWCore/Utilities"/>
+  <use name="alpaka"/>
+  <use name="HeterogeneousCore/AlpakaTest"/>
+  <use name="DataFormats/Common"/>
+  <use name="DataFormats/ParticleFlowReco"/>
+  <use name="CommonTools/ParticleFlow"/>
+  <use name="DQMServices/Core"/>
```

Comment:
Plugins needing DQM would be better to be separated from the reconstruction plugins (either a separate plugin here, or moved to a DQM package)

Reply:
There is no DQM dependencies so this was removed.